lib/looker-sdk/client/dynamic.rb

```diff
@@ -50,7 +50,21 @@ def load_swagger
         # We only need the swagger if we are going to be building our own 'operations' hash
         return if shared_swagger && @@sharable_operations[api_endpoint]
         # Try to load w/o authenticating. Else, authenticate and try again.
-        @swagger ||= without_authentication {try_load_swagger} || try_load_swagger
+        check_response = @swagger.nil?
+        @swagger ||= without_authentication { try_load_swagger }
+        response_wo_auth = last_response if check_response
```

Comment:
To be picky... you really don't need a new local variable... 
```ruby
response_wo_auth = last_response unless @swagger
```

Reply:
I suppose it doesn't matter at all in this case. But (to be pedantic and tell you why this just *seemed* kinda odd to me), in lots of universes, something called `last_response` would point to a pre-allocated and re-used buffer-ish thing containing info about the last response that would be overridden upon each request. So, hanging on to that reference and looking at it after *another* request had been made would be weird because it assumes that a new object is produced for each request. That is apparently a safe assumption here (which I assume you confirmed). So, fine.